openapi/endpoints/funding-channel/models/get-funding-channel.yaml

```
@@ -0,0 +1,22 @@
+type: object
+properties:
+  id:
+    type: string
+    description: Funding channel primary identifier
+    example: fundingChannelId1
```

```suggestion
    example: 315bad4e81ce0f26966a41b9d451638b
```

actual id use on the other endpoints would be good